src/handlers/console.ts

```diff
@@ -20,6 +20,7 @@ export class ConsoleHandler implements IDisposable {
     this.debuggerModel = options.debuggerService.model as Debugger.Model;
     this.debuggerService = options.debuggerService;
     this.consolePanel = options.widget;
+    this.consolePanel.node.setAttribute('data-jp-debugger', 'true');
```

Comment:
Should we set the attributes in both the notebook / console handlers and in the editor handler? Or would one place be enough?